github-electron/github-electron.d.ts

```diff
@@ -4643,6 +4662,11 @@ declare namespace Electron {
 		 */
 		getWebContents(): WebContents;
 		/**
+		 * Captures a snapshot of the webview's page. Same as webContents.capturePage([rect, ]callback).
+		 */
+		capturePage(callback: (image: NativeImage) => void): void;
+		capturePage(rect: Rectangle, callback: (image: NativeImage) => void): void;
```

Comment:
I would include the same description here, it's done this way in other places to keep code completion with docs working consistently for all method overrides.